src/uct/ib/rdmacm/rdmacm_cm.h

```diff
@@ -10,18 +10,24 @@
 #include <uct/base/uct_cm.h>
 #include <ucs/datastruct/khash.h>
 #include <ucs/sys/string.h>
+#include <ucs/datastruct/bitmap.h>
+#include <uct/ib/mlx5/ib_mlx5.h>
 
 #include <rdma/rdma_cma.h>
 
 
-KHASH_MAP_INIT_INT64(uct_rdmacm_cm_cqs, struct ibv_cq*);
+KHASH_MAP_INIT_INT64(uct_rdmacm_cm_qp_contexts, struct uct_rdmacm_cm_qp_context*);
```

Comment:
rename uct_rdmacm_cm_qp_contexts -> uct_rdmacm_cm_device_contexts
since it's per-device hash table  entry

Reply:
@yosefe Fixed.